src/main/java/uk/gov/digital/ho/hocs/info/api/dto/EntityDto.java

```diff
@@ -18,9 +18,15 @@ public class EntityDto {
     @JsonRawValue
     private String data;
 
+    private Boolean active;
```

Comment:
I think this could just be a primitive `boolean`, its a `not null default false` in the DB. 

Not a blocking comment though.